package Exception;

public class Q1 {

	public static void main(String[] args) {
		
		int a1 = 25;
        int a2 = 5;
        int a3 = 0;
        
        
        
        int ans1 = a1 / a2;
 
        int ans2 = a1 / a3;
 
        System.out.println("Division of va1" + " by a2 is: " + ans1);
        System.out.println("Division of va1" + " by a3 is: "+ ans2);

	}

}
